city_scrapers/spiders/ind_school_board.py

```diff
@@ -87,12 +91,17 @@ def _parse_start(self, item):
         """Parse start datetime as a naive datetime object."""
 
         title_str = item.xpath("./name/text()").extract_first()
```

Comment:
Having now run this spider and looking more closely at this code, the selectors being used to assign the value to `title_str` seem like quite a meaningful title for each of these meetings. Is there a reason we are using the hardcoded `Indianapolis Public School Board` for meeting `title` rather than this value? Assuming we're always assured of a decent meeting title in the XML, we should probably have a `_parse_title` method that extracts this info. Eg.

```
                meeting = Meeting(
                    title=self._parse_title(item)
                    ...
```
And:
```
def _parse_title(self, item):
     return item.xpath("./name/text()").extract_first()
```
Since we're using the title value to determine the classification value and start date value in the `_parse_start` and `_parse_classification` methods, we could also go one step further and retrieve the title and assign it to the Meeting object AND pass it to those methods. We would then need to modify those methods accordingly. This would DRY up the code a bit. Eg.
```
title = self._parse_title(item)
meeting = Meeting(
     title=title,
     ...
     start=self._parse_start(item, title),
     classification=self.parse_classification(item, title)
     
 ...
 
     def _parse_classification(self, item, title_str):
        ...

    def _parse_start(self, item, title_str):
        ...    
```